### DISCUSSION TITLE
IMPORT CSV With Array type doesnt load at all

### DISCUSSION THREAD

**Discussion by AntoninEfrei** [supabase/supabase#22768] [Questions]
Hey everyone.

I currently want to load a csv file that has array of strings into Supabase. Whenever I do it - it works well even in the previsualization but when i finally load the two columns that should contains array are equal to NULL 

cf : 
![image](https://github.com/supabase/supabase/assets/58555329/82856cf1-c59a-4705-9e90-e41cc2744db0)

here are my table settings : 
![image](https://github.com/supabase/supabase/assets/58555329/f130e391-d1b0-4abb-96d3-396f40a330d8)

Thanks for answering - up for any more informations required

**Comment by jorgegonzalez** [comment]
Same issue

**Comment by TicTak21** [comment]
Yeah, same problem. I thought it's a UI bug but even if I use SQL editor to query data it still returns null.

**Comment by cdaise** [comment]
Make sure your arrays are valid JSON array literals — use double quotes. It looks like you have them in your CSV as `['element 1', 'element 2', ...]` it should be `["element 1", "element 2", ...]` 

Supabase should warn you about this, it's a very easy mistake to make.

**Comment by rls129** [comment]
I had to use something like `"apple","[""banana"",""chocolate""]"`.

**Comment by r-paxst** [comment]
You can keep the format as ["bananas", "apples", "chocolate"] but the key is to find and replace your curly " with straight "s. I have no idea how to make that distinction here but you can search online for ASCII double quotes (U+0022). See below for more details. Took me forever to figure this out. smh

Straight Quotation Marks ("): These are standard ASCII double quotes (U+0022) and are recognized by PostgreSQL for text arrays.

Curly Quotation Marks (“ and ”): These are Unicode characters (U+201C and U+201D) and are not recognized as valid array delimiters in PostgreSQL.
